### ISSUE TITLE
A11y_SQLAzureDataStudioMAC_Add Database reference_Project_VoiceOver: Voiceover does not announce the radio button grouping when the VO focus lands on 'Project', 'System Database', 'Data tier application' radio button. VO announces 'Project, selected, radio button, 1 of 3'.

### ISSUE THREAD

**Issue by bbarik06** [microsoft/azuredatastudio#19884]
**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#BM_SQLAzureDataStudioMAC_June2022;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#VoiceOver;#AINotInScope;#WCAG1.3.1;

## Environment Details:
Application Name: SQL Azure Data Studio
Application Version: Version 1.37.0
Mac Version: macOS Monterey 12.4
Screen Reader: Voice-Over

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix.
2. Launch SQL Azure data studio.
3. Connect to a server.
4. Install the extension 'SQL database Projects'.
5. Tab till 'Connected server'.
6. Right click and select 'Create project from database'.
7. New database will get created.
8. Right click on data base and select "Add Database references".
9. 'Add database reference' dialog will open.
10. Tab till 'Project' radio button.
11. Turn on voiceover and observe.

## Actual:
Voiceover does not announce the radio button grouping when the VO focus lands on 'Project', 'System Database', 'Data tier application' radio button. VO announces 'Project, selected, radio button, 1 of 3'.

**The same issue can be observed throughout the application.**

## Expected:
Voiceover should announce the radio button grouping when the VO focus lands on 'Project', 'System Database', 'Data tier application' radio button. VO should announce 'Type, Project, selected, radio button, 1 of 3'. When the user leaves the last radio button VO should announce 'Group ended' and when we hit shift+ tab to the last radio button it should read the group name.

## User Impact:
Screen readers users will face inconvenience if complete information of the radio buttons is not passed to the users, and the users will not know the group that the radio buttons are part off.

## Attachment:
![Radio button group is not being announced](https://user-images.githubusercontent.com/98393966/176434395-ee47d63c-6025-4039-8969-132416b0c00f.gif)

**Comment by kisantia** [comment]
@bbarik06 this is now fixed in the latest ADS release [1.38.0](https://github.com/microsoft/azuredatastudio/releases/tag/1.38.0).

**Comment by alanrenmsft** [comment]
near SLA, so closing, please let us know if this still reproes.

**Comment by sspatil123** [comment]
GithubTags:#Closed;Version 1.41.2
![image](https://user-images.githubusercontent.com/85863999/220018058-eb3cb8c6-0271-40c5-ab30-3ac549eaaa1b.png)
